app/controllers/memberships_controller.rb

```diff
@@ -33,12 +33,11 @@ def update
     if @membership.update(membership_params)
       flash[:notice] = 'Membership Application
                         successfully updated'
-      render :show
     else
       flash[:alert] = 'A problem prevented the membership
                        application to be saved'
-      redirect_to edit_membership_path(@membership)
     end
+    redirect_to edit_membership_path(@membership)
```

Comment:
But if we take the update-status and makes it a completely different route then it should not be affected by what is in "update"?